keel-ec2-plugin/src/main/kotlin/com/netflix/spinnaker/keel/ec2/resource/SecurityGroupHandler.kt

```diff
@@ -134,11 +140,111 @@ class SecurityGroupHandler(
         .map { it.await() }
     }
 
+  override suspend fun export(exportable: Exportable): SubmittedResource<SecurityGroupSpec> {
+    val summaries = exportable.regions.associateWith { region ->
+      try {
+        cloudDriverCache.securityGroupByName(
+          account = exportable.account,
+          region = region,
+          name = exportable.moniker.name
+        )
+      } catch (e: ResourceNotFound) {
+        null
+      }
+    }
+      .filterValues { it != null }
+
+    val securityGroups =
+      coroutineScope {
+        summaries.map { (region, summary) ->
+          async {
```

Comment:
If you're using `async` here, do you want to wait for everything to complete at all?

Reply:
Ah I see the `await` at the end of this, never mind. I'm still new to these fancy things.